docs(tutorial/2 - Angular Templates): Running unit test without Firef…

risan opened the pull request:
Running unit test without having Firefox installed throws error

I am following the tutorial on AngularJS website https://docs.angularjs.org/tutorial/step_02. However I got an error when running the provided unit test without having Firefox browser installed on my machine.

Console output:

```
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
INFO [launcher]: Starting browser Firefox
ERROR [launcher]: Cannot start Firefox
    Can not find the binary /Applications/Firefox.app/Contents/MacOS/firefox-bin
    Please set env variable FIREFOX_BIN
INFO [Chrome 46.0.2490 (Mac OS X 10.11.0)]: Connected on socket zZ0c0QemlSHXzVaqVGqC with id 90305753
```

I guess it will be much clearer if we also pointed out these browsers requirement on the tutorial documentation.

risan committed:
docs(tutorial/2 - Angular Templates): Running unit test without Firefox installed throws error

I am following the tutorial on AngularJS website. However I got an error when running the provided unit test without having Firefox browser installed on my machine.

Console output:
```
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
INFO [launcher]: Starting browser Firefox
ERROR [launcher]: Cannot start Firefox
	Can not find the binary /Applications/Firefox.app/Contents/MacOS/firefox-bin
	Please set env variable FIREFOX_BIN
INFO [Chrome 46.0.2490 (Mac OS X 10.11.0)]: Connected on socket zZ0c0QemlSHXzVaqVGqC with id 90305753
``` 

I guess it will be much clearer if we also pointed out these browsers requirement on the tutorial documentation.

gkalpak commented on docs/content/tutorial/step_02.ngdoc:
```diff
@@ -195,8 +195,15 @@ to ensure that Karma and its necessary plugins are installed. You can do this by
 
-* Karma will start a new instance of Chrome browser automatically. Just ignore it and let it run in
-   the background. Karma will use this browser for test execution.
+* Karma will start new instances of Chrome and Firefox browser automatically. Just ignore them and let them run in
```
_browser_  -->  _browsers_

gkalpak commented on docs/content/tutorial/step_02.ngdoc:
```diff
@@ -195,8 +195,15 @@ to ensure that Karma and its necessary plugins are installed. You can do this by
-   the background. Karma will use this browser for test execution.
+* Karma will start new instances of Chrome and Firefox browser automatically. Just ignore them and let them run in
+   the background. Karma will use these browsers for test execution.
+* If you only have Chrome browser installed on your machine, make sure to update the karma configuration file before 
```
I think this should be more generic (e.g. what if the user has Firefox but not Chrome) ?

gkalpak commented on docs/content/tutorial/step_02.ngdoc:
```diff
@@ -195,8 +195,15 @@ to ensure that Karma and its necessary plugins are installed. You can do this by
+* Karma will start new instances of Chrome and Firefox browser automatically. Just ignore them and let them run in
+   the background. Karma will use these browsers for test execution.
+* If you only have Chrome browser installed on your machine, make sure to update the karma configuration file before 
+   running the test. Update `unit/test/karma.conf.js` file to only include Chrome browser:
```
Isn't it `test/karm.conf.js` ?

risan committed:
docs(tutorial/2 - Angular Templates): Update docs for running the unit test 

Update docs for running the unit test using karma. Explain some extra configuration to make if user only has one browser either Chrome of Firefox.

risan commented:
Thanks for the review!

Just made some updates following your comments: https://github.com/angular/angular.js/commit/647407eca9b750916fdbb1da5bbd663c87562552

gkalpak commented:
Tweaked the wording a bit (to make more concise) and merged.
Backported to `v1.4.x` as fe17c0e.
